_posts/2021-07-15-Subnero-wins-DisTec.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: Subnero wins DisTec Undersea Challenge
+banner : images/banner-pulse-standard.jpg
+date:  2021-07-15
+categories: news wnc
+thumbnail: images/pulse-thumbnail-distec.jpeg
+---
+
+We are delighted to announce that we won the [DisTec Undersea Challenge](https://www.udt-global.com/about-the-challenge){:target="_blank"}.
+
+The DisTec Undersea Challenge welcomed innovators and progressive organisations to showcase their solutions and disrupt the undersea defence environment as we know it.
+
+Our team submitted a 5 minutes [video](https://www.youtube.com/watch?v=zkaCsabBDoA&t){:target="_blank"} about the Subnero Underwater Network (SUN), which saw us selected for the Grand Finals, yesterday, along side of Marcom Defence and [Seacraft](https://seacraft.eu/){:target="_blank"}.
+
+After a 10 minutes presentation from all the finalists, we were announced as the winner of the competition.
```

Comment:
Judging was done beforehand I think, the presentation was only one of the steps.

Reply:
We don't need to mention that detail in the Pulse article, in my opinion. The set-up was to give the impression that the judges decided there and then.

Reply:
I felt it gives the impression that we won after 10 mins - sounds like an easy win.  The proposal was to just remove the 10mins part, and go with "after the presentation from..."

Reply:
Go ahead, if you think its not a big deal.

Reply:
I'm with you now, I'll amend accordingly.